Podfile.lock

```diff
@@ -197,7 +197,7 @@ SPEC CHECKSUMS:
   Gifu: 416d4e38c4c2fed012f019e0a1d3ffcb58e5b842
   Gravatar: 51437de6811c1d8d6f60c52985f2ca00a85cfc8e
   Gridicons: 4455b9f366960121430e45997e32112ae49ffe1d
-  Gutenberg: 3117b6fe578fb7f0bb75377ba96f436ee05c30fe
+  Gutenberg: cd6259542b7078fe875d51b21c962d02bac6c032
```

Comment:
Not sure why it was added – will investigate tomorrow.  When I run `pod install` in `trunk` it also changes the Podfile.lock.